openssf-compiler-options.yaml

```diff
@@ -57,6 +57,14 @@ test:
         touch foo.c
         gcc -v foo.c &>stderr.log || true
         grep gcc_s stderr.log
+    - name: Ensure etc specfile is used
+      runs: |
+        touch /etc/gcc_spec_file
+        gcc -v 2>&1 | grep "/etc/gcc_spec_file"
+    - name: Ensure GCC_SPEC_FILE is used over "/etc/gcc_spec_file"
+      runs: |
```

Comment:
can you create /etc/gcc_spec_file here ? to be clear that it exists ?

Reply:
and maybe just rm it above to make the tests more standalone.

Reply:
Can do.